faq_bot/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.10-slim-buster as base
+RUN apt update
+RUN apt install -y git
```

Comment:
```suggestion
RUN apt update & apt install -y git
```

Reply:
That will not be necessary when the framework is installed via pypi.

Reply:
Reverted in https://github.com/deeppavlov/dialog_flow_demo/pull/1/commits/07fd5466b8d564482adef972441e879222e57d9d.
Produces the error:
```
E: Unable to locate package git
The command '/bin/sh -c apt update & apt install -y git' returned a non-zero code: 100
```

Reply:
Yes, because of `RUN apt update && apt install -y git`

Reply:
https://github.com/deeppavlov/dialog_flow_demo/pull/1/commits/8c4e38b28c85bad6e02a0ec6e5b9ce074b21868a